packages/framework-core/src/booster-event-dispatcher.ts

```diff
@@ -55,27 +48,17 @@ export class BoosterEventDispatcher {
     config: BoosterConfig,
     entityName: string,
     entityID: UUID,
-    envelopes: Array<EventEnvelope>,
     eventStore: EventStore,
     readModelStore: ReadModelStore
   ): Promise<void> {
     const logger = getLogger(config, 'BoosterEventDispatcher#snapshotAndUpdateReadModels')
-    const snapshotsForIndependentEntities = await eventStore.calculateAndStoreEntitySnapshot(
-      entityName,
-      entityID,
-      envelopes
-    )
-    if (snapshotsForIndependentEntities.length === 0) {
-      logger.debug('No new snapshots generated, skipping read models projection')
+    const entitySnapshot = await eventStore.fetchEntitySnapshot(entityName, entityID)
```

Comment:
Should we rename this method as we are not snapshoting here? What do you think about `updateReadModel`?